exporters/otlp/options.go

```diff
@@ -155,3 +156,55 @@ func WithGRPCDialOption(opts ...grpc.DialOption) ExporterOption {
 		cfg.grpcDialOptions = opts
 	}
 }
+
+// ConnConfigurations keeping track of both traces and metrics
+type ConnConfigurations struct {
+	metrics config
+	traces  config
+}
+
+// DefaultConnectionOptions gives a simple way to configure the Exporter connections to default.
+var DefaultConnectionOptions []ExporterOption = []ExporterOption{
+	WithAddress(fmt.Sprintf("%s:%d", DefaultCollectorHost, DefaultCollectorPort)),
+	WorkerCount(DefaultNumWorkers),
+	WithGRPCServiceConfig(DefaultGRPCServiceConfig),
+}
+
+// NewConnections creates an empty configuration and applies
+// any ExporterOptions provided for both metrics and traces.
+func NewConnections(opts ...ExporterOption) ConnConfigurations {
```

Comment:
A big problem with the previous design was that `NewConnections` initialized both metrics and traces with default connection, resulting in broken connections if there was no collector listening at the default address. 
I think a better approach is to export the `DefaultConnectionOptions` allowing clients to specifically set them either on both connections, or on specific signals.